packages/product-json-to-csv/src/main.js

```diff
@@ -205,6 +205,8 @@ export default class ProductJsonToCsv {
       .map((price: Price) => get(price, 'channel.id'))
       .filter(Boolean)
 
+    if (!channelIds.length) return prices
```

Comment:
Nice find! Question, are we ever testing this statement?

Reply:
Here in this file with [samples](https://github.com/commercetools/nodejs/blob/1ab0df52d8554ff3d22f524a757a1b23c5f877bd/integration-tests/cli/helpers/product-json2csv.data.js#L91) we have products where one variant has a price (where we later [add a channel](https://github.com/commercetools/nodejs/pull/680/commits/1ab0df52d8554ff3d22f524a757a1b23c5f877bd#diff-e5b71a547113a8dd86fe5206592a4df3R77)) and other variants do not have prices at all so we are testing both cases.